api-specificatie/openapi.yaml

```diff
@@ -542,7 +542,7 @@ paths:
       tags:
         - Pand
       summary: "vindt panden"
-      description: "Zoek actuele panden met de identificatie van een nummeraanduiding of adresseerbaar object. Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.0.0/features/fields.feature)."
+      description: "Zoek actuele panden met de identificatie van een adresseerbaar object of een locatie. Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.0.0/features/fields.feature)."
```

Comment:
Waar is de nummeraanduiding queryparameter op panden gebleven? 
Ik zie dat in de descriptions nummeraanduidingIdentificatie voor locatie is vervangen. Ik was in de veronderstelling dat locatie er bij zou komen. Zie #271